### ISSUE TITLE
[<Error: Building package opencv:x64-windows failed with: BUILD_FAILED >] build failure

### ISSUE THREAD

**Issue by Zanardan** [microsoft/vcpkg#11802]
[config-x64-windows-out.log](https://github.com/microsoft/vcpkg/files/4738048/config-x64-windows-out.log)
**Host Environment**
- OS: [Windows 10]
- Compiler: revision

**To Reproduce**
`./vcpkg install xxxx`

**Failure logs** 
-Cut and past the appropriate build messages from the console output.
-Please attach any additional failure logs mentioned in the console output.

**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.

**Comment by timautry** [comment]
@Zanardan thank you for posting this issue. 
Please upload/attach the **`C:\vcpkg-master\buildtrees\opencv4\install-x64-windows-dbg-out.log`** file to this issue to assist in diagnosing the problem.
Thank you,

**Comment by Zanardan** [comment]
Thank you for responding. I have attached the file.

**Comment by timautry** [comment]
I do not see an obvious issue other than what is posted above.
Someone more familiar with vcpkg/OpenCV will need to assist further.
Seek advice on the Cuda question either in the Nvidia or AMD developer forums on their respective web pages as to whether Cuda supports AMD hardware.

**Comment by Zanardan** [comment]
Can you guide me how to resolve this ?
`FAILED: modules/dnn/opencv-onnx.pb.h modules/dnn/opencv-onnx.pb.cc`

**Comment by timautry** [comment]
I myself can not.
I do not use the opencv port and not familiar with it.

**Comment by JackBoosY** [comment]
```
C:\vcpkg-master\installed\x64-windows\tools\protobuf\protoc.exe --cpp_out C:/vcpkg-master/buildtrees/opencv4/x64-windows-dbg/modules/dnn -I C:/vcpkg-master/buildtrees/opencv4/src/4.1.1-fb9e10326a/modules/dnn/src/caffe -I C:/vcpkg-master/buildtrees/opencv4/src/4.1.1-fb9e10326a/modules/dnn/src/onnx -I C:/vcpkg-master/buildtrees/opencv4/src/4.1.1-fb9e10326a/modules/dnn/src/tensorflow C:/vcpkg-master/buildtrees/opencv4/src/4.1.1-fb9e10326a/modules/dnn/src/onnx/opencv-onnx.proto"
```

Can you confirm whether `libprotobuf.dll` and `libprotoc.dll` exist in _C:\vcpkg-master\installed\x64-windows\tools\protobuf_?

**Comment by Zanardan** [comment]
Yes the files: `libprotobuf.dll `and `libprotoc.dll` exist in _C:\vcpkg-master\installed\x64-windows\tools\protobuf_
I tried to install again and this is the log file obtained.  Please check this out and guide me through it.
[install-x64-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/4745576/install-x64-windows-dbg-out.log)

**Comment by JackBoosY** [comment]
@Zanardan Could you run command `cd /D C:\Dev\vcpkg\buildtrees\opencv4\x64-windows-dbg\modules\dnn && C:\Dev\vcpkg\installed\x64-windows\tools\protobuf\protoc.exe --cpp_out C:/Dev/vcpkg/buildtrees/opencv4/x64-windows-dbg/modules/dnn -I C:/Dev/vcpkg/buildtrees/opencv4/src/4.1.1-fb9e10326a/modules/dnn/src/caffe -I C:/Dev/vcpkg/buildtrees/opencv4/src/4.1.1-fb9e10326a/modules/dnn/src/onnx -I C:/Dev/vcpkg/buildtrees/opencv4/src/4.1.1-fb9e10326a/modules/dnn/src/tensorflow C:/Dev/vcpkg/buildtrees/opencv4/src/4.1.1-fb9e10326a/modules/dnn/src/tensorflow/attr_value.proto` in cmd manually?

**Comment by Zanardan** [comment]
This is the error message I am getting

![Screenshot (304)](https://user-images.githubusercontent.com/42083411/84104720-f3e2c300-aa33-11ea-8c33-ec7b4bca1507.png)

Also `attr_value.proto ` file didn't exist before, I downloaded it and pasted it there but still same.

**Comment by Cheney-W** [comment]
@Zanardan 
You could get the vcruntime140_1.dll by installing the Visual C++ Redistributable from https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads
